PREP_SYSTEM_FOR_DIETPI.sh

```diff
@@ -556,14 +563,18 @@ _EOF_
 		aPACKAGES_REQUIRED_INSTALL+=('firmware-linux-nonfree')
 		#aPACKAGES_REQUIRED_INSTALL+=('firmware-misc-nonfree')
 		#aPACKAGES_REQUIRED_INSTALL+=('dmidecode')
-		if (( $(fdisk -l $1 | grep "Disklabel type:" | awk '{ print $3 }') == 'dos' )); then
+		if [ dpkg -l | grep 'grub2 ' ]; then
```

Comment:
Following would decrease chance of unexpected scrape result :)

You probably know following already, but just if needed:
- ```^``` ensures start of line match
- ```[[:space:]]]``` is a wildcard for any amount of whitespace (eg: space, tabs)
- ```dpkg --get-selections``` same as ```dpkg -l``` without the extra ``` ii ``` stuff.
```
if [ dpkg --get-selections | grep '^grub2[[:space:]]' ]; then
```

No need to change, I'am going to add all your current PR changes to a commit my end and send. Due to all the recent global changes, PR conflict hell 😄

Reply:
Strange, on my Z83, only ```grub2-common` is installed
```
root@DietPi:~# dpkg -l | grep grub2
ii  grub2-common                   2.02~beta3-5                   amd64        GRand Unified Bootloader (common files for version 2)
```

Reply:
Jep, actually know all of this, but sometimes forget to use, was late yesterday 😆. Makes totally sense here 👍.